src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting_NetFx/DocumentFormattingTest.cs

```diff
@@ -5854,4 +5854,56 @@ void Foo() { }
                 @Foo.ToString(1)
                 """);
     }
+
+    [FormattingTestFact]
+    public async Task EscapedAtSignsInCSS()
+    {
+        await RunFormattingTestAsync(
+            input: """
+                @page "/"
+                @model IndexModel
+
+                <style>
+                    @@media only screen and (max-width: 600px) {
+                        body {
+                            background-color: lightblue;
+                        }
+                    }
```

Comment:
Should we have purposely bad formatting here to demonstrate our formatter ignores it in this case?

Reply:
Our test infra doesn't know how to format CSS or TypeScript, so it's all "bad formatting" as far as the C# formatter is concerned.

Reply:
Right, my suggestion was to have bad formatting in input an output (extra or missing space, bad indentation) so it's obvious we are not trying to format in that case. No big deal.